### PR TITLE
Docstring improvments and requirements.txt

### PR DESCRIPTION
**Pull request by sophiemachin**

### CODE REVIEW

File: thiscovery_dev_tools/testing_tools.py [hunk 1]
```diff
@@ -384,12 +384,24 @@ def test_eb_request_v2(
     **kwargs,
 ):
     """
-    Test processes triggered via EventBridge
+    Test processes triggered via EventBridge. Puts a fake event on the
```
Comment (alistairclark) [inline thiscovery_dev_tools/testing_tools.py:387]: Maybe take out "fake". It does put an event on event bridge and the event is usually one that the lambda can successfully process.

File: thiscovery_dev_tools/testing_tools.py [hunk 2]
```diff
@@ -384,12 +384,24 @@ def test_eb_request_v2(
...
+            that says the lambda finished executing.
+        - if the log is not there, check the whole log group for the log.
+            This is a time-consuming process
+    If using local method:
+        - run local method
 
     Args:
         local_method: function to be called when testing locally
-        aws_eb_event: event to be posted to event bus when testing on AWS
+        aws_eb_event: event to be posted to EventBus when testing on AWS
```
Comment (alistairclark) [inline thiscovery_dev_tools/testing_tools.py:402]: EventBridge

**Review by alistairclark — CHANGES_REQUESTED** [review]
Looking good, just a couple of things

File: thiscovery_dev_tools/testing_tools.py (hunk 2 above)
Comment (this-andy) [reply thiscovery_dev_tools/testing_tools.py:402]: or leave as original 'event bus' - both are ok

**Review by this-andy — APPROVED** [review]

File: thiscovery_dev_tools/testing_tools.py (hunk 1 above)
Comment (sophiemachin) [reply thiscovery_dev_tools/testing_tools.py]: Good point, fixed

**Comment by sophiemachin** [comment]
@alistairclark I've made those changes

**Review by alistairclark — APPROVED** [review]

**Pull request merged**